components/temoignages.js

```diff
@@ -1,16 +1,21 @@
 import PropTypes from 'prop-types'
 
 import BlogCard from '@/components/blog-card'
+import BlogPagination from '@/components/blog-pagination'
 
-function Temoignages({limit, posts}) {
+function Temoignages({limit, posts, pagination}) {
   const sortedTestimonies = limit ? posts.slice(0, limit) : posts
 
   return (
-    <div className='temoignages-section'>
-      {sortedTestimonies.map(testimony => (
-        <BlogCard key={testimony.id} post={testimony} />
-      ))}
-
+    <>
+      <div className='temoignages-section'>
+        {sortedTestimonies.map(testimony => (
+          <BlogCard key={testimony.id} post={testimony} />
+        ))}
+      </div>
+      {pagination && (
+        <BlogPagination pagination={pagination} />
```

Comment:
`BlogPagination` attend les propriétés `page`, `pages`, `next`, `prev`. Celle-ci sont contenues dans `pagination`.
```suggestion
        <BlogPagination {...pagination} />
```